packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -265,6 +291,79 @@ test('mapFileEtags returns map of S3 URIs to etags', (t) => {
   );
 });
 
+test('granuleToCmrFileObject returns correct objects for files with a bucket/key, filtering out non-CMR files', (t) => {
+  const granule = {
+    granuleId: 'fake-id',
+    files: [{
+      bucket: 'bucket',
+      key: 'fake.cmr.xml',
+    },
+    {
+      bucket: 'bucket',
+      key: 'fake.iso.xml',
+    }],
+  };
+  t.deepEqual(
+    granuleToCmrFileObject(granule),
+    [{
+      granuleId: 'fake-id',
+      bucket: 'bucket',
+      key: 'fake.cmr.xml',
+    }]
+  );
+});
+
+test('granuleToCmrFileObject returns correct objects for files with a bucket/key, filtering with a custom filter', (t) => {
+  const granule = {
+    granuleId: 'fake-id',
+    files: [{
+      bucket: 'bucket',
+      key: 'fake.cmr.xml',
+    },
+    {
+      bucket: 'bucket',
+      key: 'fake.iso.xml',
+    }],
+  };
+  const filterFunc = (fileobject) => fileobject.key.endsWith('.iso.xml');
```

Comment:
Why not use `isISOFilename` directly in this test? And update the test name to be explicit that we're testing for filtering by ISO files